service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/CaseManagementOrderEmailContentProvider.java

```diff
@@ -64,16 +66,23 @@
         CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
         return ImmutableMap.<String, Object>builder()
-            .put(SUBJECT_LINE, emailNotificationHelper
-                .buildSubjectLineWithHearingBookingDateSuffix(caseData,
-                    caseData.getHearingDetails()))
+            .put(SUBJECT_LINE, buildCallout(caseData))
             .put("respondentLastName", getFirstRespondentLastName(caseData.getRespondents1()))
             .put("digitalPreference", servingPreference == DIGITAL_SERVICE ? "Yes" : "No")
             .put(CASE_URL, servingPreference == DIGITAL_SERVICE ? getCaseUrl(caseDetails.getId()) : "")
             .putAll(linkToAttachedDocument(documentContents))
             .build();
```

Comment:
Should this not be refactored to make use of java template object?

Reply:
Out of scope for this one as it's purely about making the helper - will raise separate tech debt ticket for this though if it's not already created